### PR TITLE
Updates to change log and readme

### PR DESCRIPTION
**Pull request by yitam**
This change is [Reviewable](https://reviewable.io/reviews/microsoft/msphpsql/811)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,44 @@ All notable changes to this project will be documented in this file.
...
+- Added support for Connection Resiliency to make it available to Linux or macOS users as well (requires MS ODBC Driver 17.2)
+
+### Removed
+- Dropped support for Ubuntu 17
+
+### Fixed
+- Issue [#577](https://github.com/Microsoft/msphpsql/issues/577) - Idle Connection Resiliency doesn't work with Column Encryption enabled connection
+- Issue [#678](https://github.com/Microsoft/msphpsql/issues/678) - Idle Connection Resiliency doesn't work with Connection Pooling bug
+- Issue [#699](https://github.com/Microsoft/msphpsql/issues/699) - Binding output parameter failed when the query in the stored procedure returned no data. The test case has been added to the test lab.
+- Issue [#705](https://github.com/Microsoft/msphpsql/issues/705) - AE - Retrieving a negative decimal value (edge case) as output parameter causes truncation
```
Comment (David-Engel) [inline CHANGELOG.md]: Spell out Always Encrypted on lines 22-24.

File: README.md [hunk 2]
```diff
@@ -86,14 +86,14 @@ The version number may have trailing pre-release version identifiers to indicate
 - Build metadata may be denoted by a plus sign followed by 4 or 5 digits, such as  `1.2.3-preview+5678` or `1.2.3+5678`. Build metadata does not figure into the precedence order.
 
 ## Future Plans
-- Expand SQL Server 2016 feature support (example: Always Encrypted)
+- Expand SQL Server 2016 feature support (example: Azure AD)
```
Comment (v-mabarw) [inline README.md]: Let's spell out acronyms fully

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,41 @@ All notable changes to this project will be documented in this file.
...
+### Added
+- Added support for Azure Key Vault for Always Encrypted for basic CRUD functionalities such that Always Encrypted feature is available to all supported Windows, Linux or macOS platforms
+- Added support for macOS High Sierra (requires [MS ODBC Driver 17+](https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server?view=sql-server-2017))
+- Added support for Ubuntu 18.04 LTS (requires MS ODBC Driver 17.2)
+- Added support for Connection Resiliency to make it available to Linux or macOS users as well (requires MS ODBC Driver 17.2)
+
+### Fixed
+- Issue [#577](https://github.com/Microsoft/msphpsql/issues/577) - Idle Connection Resiliency doesn't work with Column Encryption enabled connection
+- Issue [#678](https://github.com/Microsoft/msphpsql/issues/678) - Idle Connection Resiliency doesn't work with Connection Pooling bug
+- Issue [#699](https://github.com/Microsoft/msphpsql/issues/699) - Binding output parameter failed when the query in the stored procedure returned no data. The test case has been added to the test lab.
```
[the remark this answers is not in the document]
Comment (yitam) [reply CHANGELOG.md]: Yes, @v-mabarw these tests reveal bugs in ODBC driver v17.0

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,41 @@ All notable changes to this project will be documented in this file.
...
+- Issue [#705](https://github.com/Microsoft/msphpsql/issues/705) - Always Encrypted - Retrieving a negative decimal value (edge case) as output parameter causes truncation
+- Issue [#706](https://github.com/Microsoft/msphpsql/issues/706) - Always Encrypted - Cannot insert double with precision and scale (38, 38)
+- Issue [#707](https://github.com/Microsoft/msphpsql/issues/707) - Always Encrypted - Fetching decimals / numerics as output parameters bound to PDO::PARAM_BOOL or PDO::PARAM_INT returns floats, not integers 
+- Issue [#735](https://github.com/Microsoft/msphpsql/issues/735) - Extended the buffer size for PDO lastInsertId such that data types other than integers can be supported
+- Pull Request [#759](https://github.com/Microsoft/msphpsql/pull/759) - Removed the limitation of binding a binary as inout param as PDO::PARAM_STR with SQLSRV_ENCODING_BINARY
+- Pull Request [#775](https://github.com/Microsoft/msphpsql/pull/775) - Fixed the problem for output params with SQL types specified as SQLSRV_SQLTYPE_DECIMAL or SQLSRV_SQLTYPE_NUMERIC
```
[the remark this answers is not in the document]
Comment (yitam) [reply CHANGELOG.md]: The details were explained in the pull request, mainly about the output parameters being truncated.

File: README.md (hunk 2 above)
Comment (yitam) [reply README.md]: This pull request has been merged, @v-mabarw 
If we are going to make changes, we will create another pull request. Please let us know.
